docs/testing.rst

```diff
@@ -50,3 +50,31 @@ Create your repository locally and upload that.
     $ scp -r * <fedora_username>:fedorapeople.org:/project/anaconda/repos/anaconda_fixup_repo
 
 After the above is done, everything should be fixed and Packit should work again.
+
+Fix ISO image refreshes for the WebUI tests
+-------------------------------------------
+
+In order to test the WebUI code we use the ISO images specified in `Cockpit's bots repository <https://github.com/cockpit-project/bots/tree/main/images>`_.
+
+These ISO files are direct downloads from the official distribution server and their
+`creation process is defined in bots repository <https://github.com/cockpit-project/bots/blob/main/images/scripts/fedora-rawhide-boot.bootstrap>`_.
+
+The purpose of using these ISOs for the tests instead of downloading from the official server is to gate the image updates.
+
+The default refresh period of the test images is one week, but they can also be `refreshed manually <https://github.com/cockpit-project/bots#refreshing-a-test-image>`_.
+
+Image refreshes happen with `Pull Requests which update the current image SHA <https://github.com/cockpit-project/bots/pull/2981>`_.
+The tests defined in `ui/webui/tests/` will run on the Pull request gating the `relevant to anaconda image refreshes <https://github.com/cockpit-project/bots/blob/main/lib/testmap.py>`_.
+
+In the case an updated dependency makes the WebUI tests on the image refresh fail, the on-duty team
+member is in charge of debugging the failure.
+
+For this take the following steps:
+
+    * Locally checkout the bots repository branch used in the failing refresh PR. The path to the local bots checkout should be the `ui/webui/bots`.
+    * Create the test VM with the new image and debug by following the `WebUI test instructions <https://github.com/rhinstaller/anaconda/tree/master/ui/webui/test#readme>`_
+
+When the reason for the breackage is identified there are two options to go forward:
```

Comment:
I think this should be clarified about what kinds of issues one can expect when the image refresh fails and what to do about them. Basically I see two kinds:
- an actual regression -> fill bugs for the given Fedora components, if it doesn't look like it can be fixed quickly, work around in Anaconda/mark as expected failure until fixed
- an intended change in behaviour -> adjust Anaconda or the tests
I think it would be good to reflect this somehow in this section.

Reply:
Adjusted to contain this.